ambry-rest/src/test/java/com.github.ambry.rest/NettyMultipartRequestTest.java

```diff
@@ -414,6 +428,10 @@ private void doMultipartDecodeTest(int expectedRequestSize, InMemoryFile[] files
     request.readInto(asyncWritableChannel, null).get();
     readOutput = asyncWritableChannel.getData();
     assertArrayEquals(RestUtils.MultipartPost.BLOB_PART + " content does not match", blobData.array(), readOutput);
+    byte[] partByPartDigest = request.getDigest();
+    if (partByPartDigest != null) {
```

Comment:
You are right. I removed it in most of the other places. Missed this one. Thanks !